modules/015-admission-policy-engine/hooks/bootstrap_handler.go

```diff
@@ -57,31 +63,78 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 
 func handleGatekeeperBootstrap(input *go_hook.HookInput) error {
 	var bootstrapped bool
-	var existingTemplates = set.NewFromSnapshot(input.Snapshots["gatekeeper_templates"])
+	templates := input.Snapshots["gatekeeper_templates"]
 
-	if existingTemplates.Size() != 0 {
+	if len(templates) != 0 {
+		existingTemplates := make(map[string]struct {
```

Comment:
you can reuse `cTemplate` here, not to create a new struct, just don't fill the `Name` field